### ISSUE TITLE
[Catalog] Buttons FAB demo announces icon name in VoiceOver

### ISSUE THREAD

**Issue by ianegordon** [material-components/material-components-ios#3164]
The BottomAppBar demo FAB is missing an accessibilityLabel and instead reads out the icon image name.  Instead, it should have something generic like, "Primary action".

---

Related internal bug: b/74075419

---
#### Internal data
- Associated internal bug: [b/117178915](http://b/117178915)

**Comment by romoore** [comment]
That internal issue cannot be resolved by fixes in MDC-iOS. It's a client app that needs to set its own accessibility labels for the FAB.

**Comment by romoore** [comment]
I'm not actively working on this.

**Comment by romoore** [comment]
Closing this due to inactivity greater than 6 months. Please feel free to reopen if you believe it should be worked on in the near future.
